workitem/enum_type.go

```diff
@@ -40,6 +40,11 @@ var _ convert.Equaler = (*EnumType)(nil)
 
 // Equal returns true if two EnumType objects are equal; otherwise false is returned.
 func (t EnumType) Equal(u convert.Equaler) bool {
+	return t.DefaultEqual(u)
```

Comment:
Is that really needed or can we get rid of `DefaultEqual` now?

Reply:
I kept in in there because I still think to have an EqualEnclosing() available is a good idea. We might need that sometimes.

Reply:
Oh my point was to just include the `DefaultEqual` code here directly since it is nowhere else used and `Equal` is per-se the default-Equal IMHO.

Reply:
Ok, fixed.